setup-gems-cache/action.yml

```
@@ -0,0 +1,32 @@
+name: Setup Gems Cache
+description: Setup Gems Caching for workflow
+inputs:
+  gems-path:
+    description: Homebrew's Ruby gems path
+    required: true
+  gems-hash:
+    description: Homebrew's Ruby Gemfile.lock sha256sum
+    required: true
+outputs:
+  gems-outcome:
+    description: "Brew Install Bundler Gems steps outcome"
+    value: ${{ steps.gems.outcome }}
+
+runs:
+  using: composite
+  steps:
+    - name: Cache Homebrew Gems
+      id: cache
+      uses: actions/cache@v3
+      with:
+        path: ${{ inputs.gems-path }}
+        key: ${{ runner.os }}-rubygems-${{ inputs.gems-hash }}
+        restore-keys: ${{ runner.os }}-rubygems-
+
+    - name: Install Homebrew Gems
+      id: gems
+      run: |
+        # Install Homebrew Gems
+        brew install-bundler-gems
```

I don't mind this verbosity, but it's probably not necessary and can become a one-liner

```suggestion
      run: brew install-bundler-gems
```

---

I think you're right. We can see the purpose of the command just by reading it.